content/docs/command-reference/fetch.md

```diff
@@ -90,9 +90,9 @@ or `-T` options are used).
 
 - `-j <number>`, `--jobs <number>` - number of threads to run simultaneously to
   handle the downloading of files from the remote. The default value is
-  `4 * cpu_count()`. For SSH remotes, the default is just `4`. Using more jobs
-  may improve the total download speed if a combination of small and large files
-  are being fetched.
+  `4 * cpu_count()`. For SSH remotes, the default is `4`. Using more jobs may
+  improve the total download speed if a combination of small and large files are
```

Comment:
no sure about this note - why only combination?

Reply:
I didn't cange this note, only removed the word "just" for consistency with a corresponding change you made in `status`.

Reply:
I see, yes, let's then remove it as well. (`just` is already removed btw)

Reply:
> Using more jobs may improve the total download speed if a combination of small and large files are being fetched.

I think this was written by the engineering team a while back. (maybe by @pared ?) So I just accepted it as truth.

Reply:
But anyway. Removed the note and standardized this option desc. for all refs. in 80a0f092.

Reply:
thanks! we can probably keep generic explanation like "Using more jobs may improve the total download speed."

there are a lot of situations when that might happen (high latency connection, qos per connection on both ends). Curios about large and small files combination though- may be I'm missing something.

Reply:
Added "Using more jobs may improve the overall transfer speed." to all for now 🙂 

> Curios about large and small files combination

I think it means one big file is downloaded in a thread while many small ones are downloaded in the other ones so somehow there's no bottle neck... But I'm also a bit skeptical of this logic now that you mention it.

Reply:
@jorgeorpinel That might have been me, though I do not remember this one. Anyway, I could agree that it might get faster for any combination of files. Using multiple jobs parallelizes tasks such as opening file descriptors, so even in case of limited connection speed, it might help (to some extent). Anyway, I don't think its something user might want to spend time on, and such inclusion hinders understanding this option.

Reply:
Thanks for the comment Pawel!